locales/de.json

```diff
@@ -1,574 +1,911 @@
+    },
+    "aria-allowed-attr": {
+      "description": "",
+      "help": "Elemente dürfen nur erlaubte ARIA-Attribute verwenden."
+    },
+    "aria-allowed-role": {
+      "description": "Der Wert des role-Attributes muss gültig sein.",
+      "help": "Der Wert des role-Attributes muss für dieses Element geeignet sein."
+    },
+    "aria-command-name": {
+      "description": "Stellt sicher, dass jede ARIA-button, jeder -link und jeder -menuitem einen barrierefreien Namen hat.",
+      "help": "ARIA Befehle müssen einen barrierefreien Namen besitzen."
+    },
+    "aria-dialog-name": {
+      "description": "Stellt sicher, dass jeder ARIA-dialog und -alertdialog Knoten einen barrierefreien Namen hat.",
+      "help": "ARIA-dialog und -alertdialog Knoten müssen einen barrierefreien Namen besitzen."
+    },
+    "aria-hidden-body": {
+      "description": "",
+      "help": "Aria-hidden='true' darf nicht für den <body> des Dokumentes verwendet werden."
+    },
+    "aria-hidden-focus": {
+      "description": "Stellt sicher, dass ARIA-hidden Elemente keine fokussierbaren Elemente beinhalten.",
+      "help": "ARIA-hidden Elemente dürfen keine fokussierbaren Elemente beeinhalten."
+    },
+    "aria-input-field-name": {
+      "description": "Stellt sicher, dass jedes ARIA-inputs einen barrierefreien Namen besitzt.",
+      "help": "ARIA-inputs müssen einen barrierefreien Namen besitzen."
+    },
+    "aria-meter-name": {
+      "description": "Stellt sicher, dass jeder ARIA-meter Knoten einen barrierefreien Namen besitzt.",
+      "help": "ARIA-meter Knoten müssen einen barrierefreien Namen besitzen."
+    },
+    "aria-progressbar-name": {
+      "description": "Stellt sicher, dass jeder ARIA-progressbar Knoten einen barrierefreien Namen besitzt.",
+      "help": "ARIA-progressbar Knoten müssen einen barrierefreien Namen besitzen."
+    },
+    "aria-required-attr": {
+      "description": "",
+      "help": "Erforderliche ARIA-Attribute müssen bereitgestellt werden."
+    },
+    "aria-required-children": {
+      "description": "",
+      "help": "Bestimmte ARIA-Rollen müssen spezifische, untergeordnete Kind-Rollen enthalten."
+    },
+    "aria-required-parent": {
+      "description": "",
+      "help": "Bestimmte ARIA-Rollen müssen in spezifischen, übergeordneten Eltern-Rollen enthalten sein."
+    },
+    "aria-roledescription": {
+      "description": "Stellt sicher, dass ARIA-roledescription nur im Zusammenhang mit einer im- oder expliziten Rolle verwendet wird.",
+      "help": "Nutze aria-roledescription für Elemente mit einer semantischen Rolle."
+    },
+    "aria-roles": {
+      "description": "",
+      "help": "Verwendete ARIA-Rollen müssen gültigen Werten entsprechen."
+    },
+    "aria-toggle-field-name": {
+      "description": "Stellt sicher, dass jedes ARIA-toggle-Feld ein barrierefreien Namen besitzt.",
+      "help": "ARIA-toggle-Felder besitzen einen barrierefreien Namen."
```

Comment:
Would `ARIA-toggle-Felder muessen einen barrierefreien Namen besitzen.` not be more consistent with the other help texts?

Reply:
or `ARIA-toggle-Felder benötigen einen barrierefreien Namen.` as below